grl/environment/examples_lib.py

```diff
@@ -83,9 +98,13 @@ def example_11():
     ])
 
     Pi_phi = [
-        np.array([0, 0, 0, 0]),
+        np.array([
+            [1],
+            [1],
+            [1]
+        ]),
```

Comment:
Just so I'm clear... this means 100% probability of executing the only action there is, for all three possible observations, right?

Reply:
Correct